rootfs/etc/nginx/lua/balancer/static_balancers.lua

```diff
@@ -0,0 +1,78 @@
+local split = require('util.split')
+local ngx_upstream = require("ngx.upstream")
+local implementations = require('balancer.implementations')
+
+local backends = {}
+local balancers = {}
+
+local DEFAULT_LB_ALG = "ewma"
+
+local _M = {}
+
+local function parse_endpoint(endpoint)
+    if (not endpoint.address) or (not endpoint.port) then
+        if endpoint.addr then
+            local addr, err = split.parse_addr(endpoint.addr)
+            if err then
+                return nil, err
+            end
+
+            endpoint.address = addr.host
+            endpoint.port = addr.port
+            endpoint.addr = nil
+
+            return endpoint, nil
+        end
+    end
+    return nil, "error in grabbing address & port"
+end
+
+local function create_backend(upstream_name)
+    local sb = {
+        name = upstream_name,
+        endpoints = {},
+        ['load-balance'] = DEFAULT_LB_ALG
```

Comment:
lets stick to `"` for strings.

and I liked your idea of specifying load balancing alg as an argument to `configure()`

Reply:
> lets stick to " for strings.

Seems like is more natural to use double quotes for user visible strings and text, whatever appears in output or in files; and single quotes for option-like strings, passing to methods and generally in code. 

Can change if you feel strongly about this. 

> and I liked your idea of specifying load balancing alg as an argument to configure()

Sure, will do this in a subsequent PR.